ui/perfherder/RevisionInformation.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { react2angular } from 'react2angular/index.es2015';
+import { ListGroup, ListGroupItem } from 'reactstrap';
+
+import perf from '../js/perf';
+
+import { getRevisionUrl } from './helpers';
+
+function getRevisionSpecificDetails(
+  revision,
+  project,
+  isBaseline,
+  resultSet,
+  selectedTimeRange = null,
+) {
+  const revisionUrl = getRevisionUrl(revision, project.name);
+  const truncatedRevision = revision.substring(0, 12);
+  let baselineOrNew = isBaseline ? 'Base' : 'New';
+  if (selectedTimeRange) baselineOrNew = 'Base';
```

Comment:
Another way of writing these two lines might be:

```js
let baselineOrNew = isBaseline || selectedTimeRange ? 'Base' : 'New';
```

Reply:
Nifty!